suites/upgrade/infernalis/older/2-workload/blogbench.yaml

```diff
@@ -0,0 +1,11 @@
+meta:
+   - desc: |
+      run a filesystem stress test
+      (not running on cephfs)
```

Comment:
need to fix this to run - ceph-fuse on client.0 before the workunit. As is, it's not using ceph, and just wasting machine time